uswds/uswds-site
# USWDS-Site - Card: Fix card grid presentation

# Summary
- **Switched out the grid classes in the card preview template.** This improves the appearance of the card component preview at narrow widths.
- **Added grid styles specifically suited for the card component preview.**
- **Added a changelog entry for card grid fix.**

## Related issue

Related to https://github.com/uswds/uswds-site/issues/2323

## Preview link

- [Card component preview](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-card-grid-fix/components/card/#card-preview-content)
- [Card changelog ](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-card-grid-fix/components/card/#latest-updates)

## Problem statement

The [card component preview page](https://designsystem.digital.gov/components/card/) shows cards that get too narrow and elements within the cards overlap at narrow browser widths. 

![image](https://github.com/uswds/uswds/assets/93996430/829f2f51-e5ee-410c-94cf-83bd3863cdde)

## Solution
Adjusting the grid breakpoints in the card template improves the display of the component at narrow widths. 

This fix currently provides specific grid classes for only for the scenarios needed. Alternatively, we could turn on the "widescreen" and "tablet-lg" values in `$theme-utility-breakpoints`. However, this felt heavy-handed for this specific need (See more info in [this comment](https://github.com/uswds/uswds-site/pull/2299#issuecomment-1819706789)). 

## Testing and review
- Confirm the [card component preview](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-card-grid-fix/components/card/#card-preview-content) presents well at various breakpoints and has the correct grid breakpoints. 
- Confirm that the approach for updating the styles makes sense. 
- Confirm the changelog is accurate and free from spelling and grammatical errors.
